Presentation.Web/Controllers/OData/OrganizationRightsController.cs

```diff
@@ -88,7 +88,7 @@ public IHttpActionResult PostRights(int orgKey, OrganizationRight entity)
         /// <returns></returns>
         public override IHttpActionResult Post(OrganizationRight entity)
         {
-            return Unauthorized();
+            return StatusCode(HttpStatusCode.NotImplemented);
```

Comment:
Hvorfor "Not implemented"? Måske er hensigten med override at gøre det eksplicit at man ikke kan poste på en OrganizationRightsController? I modsat fald er det måske bedre helt at fjerne metoden?